src/Microsoft.AspNetCore.Razor.Language/RazorEngine.cs

```diff
@@ -15,26 +15,39 @@ public static RazorEngine Create()
             return Create(configure: null);
         }
 
-        public static RazorEngine Create(Action<IRazorEngineBuilder> configure)
+        public static RazorEngine Create(Action<IRazorEngineBuilder> configure) => Create(RazorLanguageVersion.Latest, configure);
+
+        public static RazorEngine Create(RazorLanguageVersion version, Action<IRazorEngineBuilder> configure)
```

Comment:
Thoughts on these being `public`? At first I had them `internal` but then I figured that providing the `RazorLanguageVersion` is pretty essential to building a `RazorEngine`. Given that we'll have a similar API for the `RazorProjectEngine` (but will exist on the builder) I wanted to keep a similar API available even if you can't detect it directly via the builder.